src/components/molecules/NavSearchBar/NavSearchBar.scss

```diff
@@ -1,8 +1,19 @@
 @import "scss/constants.scss";
 
+$search-results-width: 18.125rem;
+
 .nav-search-links {
-  display: flex;
```

Comment:
Why did we remove this?

Reply:
I assumed that it's not needed anymore. Because its children are: div->input and search-result (which has fixed position and flex doesn't apply to it).
should I give `flex` back?

Reply:
Nope, if it's not needed it's fine to remove. Just wanted to make sure we were doing so consciously/etc.